broker-cli/proto/broker.proto

```diff
@@ -859,6 +861,10 @@ service OrderBookService {
   rpc WatchMarket (WatchMarketRequest) returns (stream WatchMarketResponse);
   /**
    * GetOrderbook returns price and amount information about the bids and asks on the orderbook in the current state.
+   *
+   * When GetOrderbookRequest includes a limit parameter, the total number of bids or asks will not exceed
```

Comment:
hmm, it's not the total number of bids and asks, its that the number of bids will not exceed the limit and the number of asks will not exceed the limit. Right?